cfy_manager/components/restservice/config/authorization.conf

```diff
@@ -650,6 +650,34 @@ permissions:
     - operations
     - viewer
 
+  filters_create:
+    - sys_admin
+    - manager
+    - user
+    - operations
+    - viewer
+
+  filters_list:
+    - sys_admin
+    - manager
+    - user
+    - operations
+    - viewer
+
+  filters_get:
+    - sys_admin
+    - manager
+    - user
+    - operations
+    - viewer
+
+  filters_update:
+    - sys_admin
+    - manager
+    - user
+    - operations
+    - viewer
+
```

Comment:
I didn't think there is a need to create a different rule for `filters_delete` as it has the same behavior as filters_update.